### PR TITLE
PEP 8 fixes

### PR DESCRIPTION
**Pull request by pashields**
Some tiny PEP8 fixes for setup.py. I'm not particularly worried about setup.py following PEP8 in general, but this will stop my editor from complaining!

### CODE REVIEW

**Comment by melaniedc** [comment]
+1

File: setup.py [hunk 1]
```diff
@@ -26,9 +26,10 @@ def parse_version(module_file):
 ga4gh_readme = f.read()
```
Comment (jeromekelleher) [inline setup.py]: Can we camelCase these as well please?

**Comment by kerrydc** [comment]
+1 with Jerome's suggestion

**Comment by pashields** [comment]
requested formatting is in.

File: setup.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 # Following the recommendations of PEP 396 we parse the version number
 # out of the module.
-def parse_version(module_file):
+def parseVersion(module_file):
```
Comment (jeromekelleher) [inline setup.py]: Can we change `module_file` also please?

File: setup.py [hunk 3]
```diff
@@ -23,22 +23,23 @@ def parse_version(module_file):
...
+ga4ghReadme = f.read()
 f.close()
-ga4gh_version = parse_version("ga4gh/__init__.py")
+ga4ghVersion = parseVersion("ga4gh/__init__.py")
 # Flask must come after all other requirements that have "flask" as a prefix
 # due to a setuptools bug.
-requirements = ["avro", "Flask-API", "flask-cors", "flask", "pysam", "requests", "wormtable"]
+requirements = ["avro", "Flask-API", "flask-cors", "flask", "pysam",
+                "requests", "wormtable"]
 v = sys.version_info[:2]
```
Comment (jeromekelleher) [inline setup.py]: Maybe we should delete these 3 lines also? I doubt we're every going to support Python 2.6 or 3.1 (TravisCI have stopped supporting 3.1) and we definitely don't need to worry about supporting 3.0. Sorry, I know this was just a quick pep8 update - feel free to ignore this if you don't want to deal with this now!

**Comment by pashields** [comment]
Sorry for the delay. made @jeromekelleher's suggested fixes. I'd like to cut this off and merge from here though.

**Comment by jeromekelleher** [comment]
Thanks @pashields, sorry for loading a bunch of extra stuff in there!

+1.

**Pull request merged**